function a() {
    try {

    } catch(e) {
        return 34;
    }

    try {

    } catch(e) {
        console.log('teste')
        return 2;
    }

    try {

    } catch(e) {
        return 'error';
    }

    try {

    } catch(e) {
        return null;
    }

    while(true) {
        try {

        } catch (e) {
            return undefined;
        }
    }

    try {

    } catch (e) {
        console.log('teste')
        return undefined;
    }

}

